src/templates/pycontw-2020/_includes/portal.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+
+<div class="card">
+	<a class="button-round" href="{% url 'events_schedule' %}" class="card-link">{% trans 'Schedule' %}</a>
+</div>
+
+<div class="card">
+	<a class="button-round" href="https://hackmd.io/@PyConTW/2019" class="card-link">{% trans 'Collaborative Notes' %}</a>
+</div>
+
+<div class="card">
+	<a class="button-round" href="http://t.me/pycontw2018" class="card-link">{% trans 'Chat Room' %}</a>
```

Comment:
Currently it is `http://t.me/pycontw2018` but we are going to use `http://t.me/pycontw`

Reply:
We could keep this link until we do update the link of telegram. Please keep contacting with me.

Reply:
I got it , I though `http://t.me/pycontw2018` is working , so I didn't edit it , okay I'll update to  `http://t.me/pycontw` .

Reply:
No. Sorry for confusing. Please keep the original link and I will update later when it's the link migration date.